File: Kotlin/FizzBuzz.kt
fun main() {
    for (counter in 1..100) {
        when {
            counter % 15 == 0 -> println("FizzBuzz")
            counter % 5 == 0 -> println("Buzz")
            counter % 3 == 0 -> println("Fizz")
            else -> println(counter)
        }
    }
}

File: Dart/FizzBuzz.dart
void main() {
  for (var counter = 1; counter <= 100; counter++) {
    if (counter % 3 == 0) {
      if (counter % 5 == 0) {
        print('FizzBuzz');
      } else {
        print('Fizz');
      }
    } else if (counter % 5 == 0) {
      print('Buzz');
    } else {
      print(counter);
    }
  }
}
